Muehle_GUI.ipynb

```diff
@@ -260,19 +274,26 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "class Status():\n",
-    "    state = [[9,9],[          # Anzahl zu setzender Steine (Spieler_1 (Weiß), Spieler_2 (Braun))\n",
-    "            [0,0,0,0,0,0,0,0], # ring 0\n",
-    "            [0,0,0,0,0,0,0,0], # ring 1\n",
-    "            [0,0,0,0,0,0,0,0]  # ring 2\n",
-    "      ]]\n",
-    "    current_player = 1\n",
-    "    mills = set()              # Tupel der Form (Spieler, Ring, Zelle)\n",
-    "    number_stones_to_remove = 0\n",
-    "    selected_dot_old = (None, None)\n",
-    "    selected_dot_new = (None, None)\n",
-    "    move_or_jump = False\n",
-    "    game_over = False"
+    "class Game():\n",
+    "    def __init__(self, state = copy.deepcopy(START_STATE), current_player = 1):\n",
+    "        self.state = state\n",
+    "        self.current_player = current_player\n",
+    "        self.mills = set()\n",
+    "        self.number_stones_to_remove = 0\n",
+    "        self.selected_dot_old = (None, None)\n",
+    "        self.selected_dot_new = (None, None)\n",
+    "        self.move_or_jump = False\n",
+    "        self.game_over = False\n",
+    "    \n",
+    "    def __str__(self):\n",
+    "        return \"state: \" + str(self.state) \\\n",
+    "            + \"\\ncurrent player: \" + str(self.current_player) \\\n",
+    "            + \"\\nmills: \" + str(self.mills) \\\n",
+    "            + \"\\nnumber stones to remove: \" + str(self.number_stones_to_remove) \\\n",
+    "            + \"\\nselected dot old: \" + str(self.selected_dot_old) \\\n",
+    "            + \"\\nselected dot new: \" + str(self.selected_dot_new) \\\n",
+    "            + \"\\nmove or jump: \" + str(self.move_or_jump) \\\n",
+    "            + \"\\ngame over: \" + str(self.game_over) "
```

Comment:
In Verbindung hiermit funktioniert die aktuelle implementierung des restart-Button nicht => kann man ja in nem extra issue lösen

Reply:
Oh, was funktioniert nicht? Bei mir hatte sich meine ich alles zurück gesetzt, aber es ist noch ein zweites Feld darunter aufgetaucht. Ich schau es mir nochmal an und mache im Zweifelsfall ein Issue auf

Reply:
Bei mir ist dann einfach nichts passiert

Reply:
die Funktion 
![grafik](https://user-images.githubusercontent.com/43981971/109478256-4d36ca80-7a79-11eb-9d30-48328830944e.png)

Reply:
Probier's jetzt nochmal. Jetzt geht es bei mir. Aber es wird immer noch untendrunter ein neues Feld angelegt. Da habe ich grade keine Idee wie man das lösen kann und würde ein Bug-Issue anlegen

Reply:
Bug-Issue für Restart-Button: #68

Reply:
Bei mir wird kein extra Feld erzeugt